packages/presets/src/ai/utils/selection-utils.ts

```diff
@@ -250,9 +250,8 @@ export const getSelectedImagesAsBlobs = async (host: EditorHost) => {
     .run();
 
   const blobs = await Promise.all(
-    data.currentBlockSelections?.map(async s => {
-      const sourceId = (host.doc.getBlock(s.blockId)?.model as ImageBlockModel)
-        ?.sourceId;
+    data.selectedBlocks?.map(async b => {
```

Comment:
what is the difference between selectedBlocks & currentBlockSelections?

Reply:
selectedBlocks includes currentBlockSelections and currentImageSelections, current implementation would return null when there are only image selections.